.github/workflows/tld-update.yml

```diff
@@ -0,0 +1,46 @@
+name: tld-update
+on:
+  schedule:
+    # Run every hour, at the 15 minute mark. E.g.
+    # 2020-11-29 00:15:00 UTC, 2020-11-29 01:15:00 UTC, 2020-11-29 02:15:00 UTC
+    - cron:  '15 * * * *'
+jobs:
+  zlint-gtld-update:
+    name: Check for TLD data updates
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.15
+
+      - name: Set current date as env variable
+        run: echo "NOW=$(date +'%Y-%m-%dT%H:%M:%S %Z')" >> $GITHUB_ENV
+
+      - name: Install zlint-gtld-update
+        run: go install ./cmd/zlint-gtld-update/...
+        working-directory: v3
+
+      - name: Run go-generate
+        run: go generate ./...
+        working-directory: v3
+
+      - name: Create pull-request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: "util: gtld_map autopull updates for ${{ env.NOW }}"
+          title: "util: gtld_map autopull updates for ${{ env.NOW }}"
+          body: "ZLint gTLD data updates from `go generate ./...` for ${{ env.NOW }}."
+          labels: tld-update
+          branch: zlint-gtld-update
```

Comment:
What happens if we have two unmerged PRs? Do we want to use the same branch name every time?

Reply:
I haven't tested this part yet (it's trickier to arrange) but my understanding is that the bot will continue to append commits to the open PR until it's merged.

The action author has a longer description of how this works here: https://github.com/peter-evans/create-pull-request#action-behaviour

Reply:
The "Alternative strategy - Always create a new pull request branch" is closer to what we had before and it's not very good for this use-case.

Reply:
That sounds reasonable to me -- mostly just wanted to make sure wouldn't crash.

Reply:
Happy to iterate further based on our experience if bugs show up.